examples/visualization/styles/helpers/color-bins-style.html

```diff
@@ -49,12 +49,12 @@
         // breaks: [0, 100, 200, 300, 365],
         // Methods allowed are quantiles, equal and stdev
         method: 'quantiles',
-
-        // opacity: 0.5
-
-        // palette accepts an array of HEX color strings,
-        // or a string pointing to a CARTOColors palette
-        // palette: 'teal'
+        
```

Comment:
I have tried here with bins, and it works if the value is low, but we must cap the max number of bins, because if it is high (eg 20), we run out of colors when getting it from the 'teal' palette and it fails on hexToRgb function